pytapo/__init__.py

```diff
@@ -563,7 +563,13 @@ def performRequest(self, requestData, loginRetryCount=0):
             encryptedResponse = responseData["result"]["response"]
             encryptedResponse = base64.b64decode(responseData["result"]["response"])
             try:
-                responseJSON = json.loads(self.decryptResponse(encryptedResponse))
+                decodedResponse = self.decryptResponse(encryptedResponse).decode("utf-8")
```

Comment:
I wonder why this was not needed on other cameras, I will run some tests and check code later to see if it doesn't cause any issues.